src/openforms/logging/templates/logging/events/outgoing_request_log_details_view_admin.txt

```diff
@@ -0,0 +1,4 @@
+{% load i18n %}
+{% blocktrans trimmed with lead=log.fmt_lead user=log.fmt_user requestlog=log.content_object %}
+   {{ lead }}: User {{ user }} viewed outgoing request log {{ requestlog.method }} {{ requestlog.url }} in the admin
```

Comment:
this syntax is broken - try it out in your local environment

you can't use lookups inside `blocktrans`